captura/test_api_views.py

```diff
@@ -401,11 +401,11 @@ def test_remove_integrante(self):
         study_id = response.data['id']
         change_study = response.data
         change_study['familia']['integrante_familia'][0]['activo'] = False
-        integrante_id = change_study['familia']['integrante_familia'][0]['id']
+        id_integrante = change_study['familia']['integrante_familia'][0]['id']
```

Comment:
i just changed this because we are always using id_integrante. I messed up in the form putting integrante_id and then replaced all the  integrante_id with id_integrante.